src/PostgreSQLCopyHelper.NodaTime.Test/Extensions/NpgsqlExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Npgsql;
+
+namespace PostgreSQLCopyHelper.NodaTime.Test.Extensions
+{
+    public static class NpgsqlExtensions
```

Comment:
I think we should pull this from `PostgreSQLCopyHelper.Test/Extensions/NpgsqlExtensions.cs` instead of making another copy of it.

Reply:
Reasonable.